src/test/java/service/CsvFileWriterServiceTest.java

```diff
@@ -0,0 +1,80 @@
+package service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvFileWriterServiceTest {
+    private static FileWriterService fileWriterService;
+    private static final String PATH = "src" + File.separator
+                                    + "test" + File.separator
+                                    + "java" + File.separator
+                                    + "resources";
+    private static final File TEST_FILE =
+            new File(PATH + File.separator + "testFile.csv");
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileWriterService = new CsvFileWriterService();
+    }
+
+    @After
+    public void tearDown() {
+        TEST_FILE.delete();
+    }
+
+    @Test
+    public void saveToFile_someData_Ok() {
+        List<String> expected = new ArrayList<>();
+        String data;
+        for (int i = 1; i <= 10; i++) {
+            data = i + " operation, fruit, amount";
+            expected.add(data);
+        }
+        File resultFile = fileWriterService.saveToFile(TEST_FILE, expected);
+        List<String> actual = readFile(resultFile);
+        Assert.assertEquals("Result is incorrect", expected, actual);
+    }
+
+    @Test
+    public void saveToFile_emptyData_Ok() {
+        List<String> expected = new ArrayList<>();
+        File resultFile = fileWriterService.saveToFile(TEST_FILE, expected);
+        List<String> actual = readFile(resultFile);
+        Assert.assertEquals("Result is incorrect", expected, actual);
+    }
+
+    @Test
+    public void saveToFile_nullData_Ok() {
+        File resultFile = fileWriterService.saveToFile(TEST_FILE, null);
+        List<String> expected = new ArrayList<>();
+        List<String> actual = readFile(resultFile);
+        Assert.assertEquals("Result is incorrect", expected, actual);
+    }
+
+    @Test
+    public void saveToFile_nullFile_NotOk() {

```

Comment:
the same, fix everywhere
(@Test(expected = RuntimeException.class))

Reply:
It works, thank`s. So much better now!